www/servers.html

```diff
@@ -53,7 +53,8 @@ <h3>Roubaix-fr.pirateirc.net</h3>
       <strong>staff</strong>: <a href="mailto:mikaela.suomalainen@piraattipuolue.fi">Mikaela</a> (PGP: 2910 4A46 C561 5BF9 78A0 83F2 0C20 7F07 B2F3 2B67), <a href="mailto:ari-martti.hopiavuori@piraattipuolue.fi">AriMartti</a>, <a href="mailto:scoffa@pirateirc.net">Scoffa</a></br>
       <strong>features</strong>: IPv4, SSL (commercial / verifiable), SASL, Tor, Yggdrasil</br>
       <ul>Alternative connection points
-      <li><strong>Tor</strong>: <code>tll4bxf546kzf6iv4n2m4pbbjnifrfewe3kcritva2tuuuiowygx2cqd.onion</code></li>
+      <li><strong>Tor</strong>: <code>tll4bxf546kzf6iv4n2m4pbbjnifrfewe3kcritva2tuuuiowygx2cqd.onion</code>
```

Comment:
I think this could be done better, but it's outside of my abilities, so I would welcome help.